Dockerfile

```diff
@@ -7,7 +7,9 @@ ARG TAG
 RUN apk add --no-cache git
 
 COPY --from=src-cli /usr/bin/src /usr/bin
+
+RUN npm i -g n
```

Comment:
Why `n` vs `nvm`? (I've not had experience with n)

Reply:
![image](https://user-images.githubusercontent.com/18282288/105895792-de93c680-600d-11eb-82f2-0bb2fdd5c74b.png)
Handles all/most of the other tools configs in one

Reply:
oh rad.